### PR TITLE
desired value update

### PR DESCRIPTION
**Pull request by bretf-ik**

### CODE REVIEW

File: src/SmartHotel.Clients.sln [hunk 1]
```diff
@@ -304,36 +304,36 @@ Global
 		{0D0C7A31-009A-4A19-9663-0F99791FAAC5}.Release|x86.ActiveCfg = Release|x86
 		{0D0C7A31-009A-4A19-9663-0F99791FAAC5}.Release|x86.Build.0 = Release|x86
 		{0D0C7A31-009A-4A19-9663-0F99791FAAC5}.Release|x86.Deploy.0 = Release|x86
-		{0D0C7A31-009A-4A19-9663-0F99791FAAC5}.UI Test|Any CPU.ActiveCfg = Release|Win32
-		{0D0C7A31-009A-4A19-9663-0F99791FAAC5}.UI Test|Any CPU.Build.0 = Release|Win32
-		{0D0C7A31-009A-4A19-9663-0F99791FAAC5}.UI Test|Any CPU.Deploy.0 = Release|Win32
+		{0D0C7A31-009A-4A19-9663-0F99791FAAC5}.UI Test|Any CPU.ActiveCfg = Debug|x64
```
Comment (willyt150) [inline src/SmartHotel.Clients.sln]: Can you please undo these changes to the .sln file?

**Comment by bretf-ik** [comment]
Done

Bret Faller  | InterKnowlogist | Lead Software Engineer | Mobile 858.243.1671
IK<http://www.interknowlogy.com/> | Vimeo<https://vimeo.com/interknowlogy/> | LinkedIn<https://www.linkedin.com/company/44716?trk=vsrp_companies_cluster_name&trkInfo=VSRPsearchId%3A227721001456353449886%2CVSRPtargetId%3A44716%2CVSRPcmpt%3Acompanies_cluster> | Twitter<https://twitter.com/Interknowlogy> | Facebook<https://www.facebook.com/InterKnowlogy>

//interknowlogy/ a think tank of makers who create for all digital surfaces<http://www.interknowlogy.com/>

File: src/SmartHotel.Clients.sln [hunk 2]
```diff
@@ -316,7 +316,7 @@ Global
 		{0D0C7A31-009A-4A19-9663-0F99791FAAC5}.UI Test|iPhoneSimulator.ActiveCfg = Release|Win32
 		{0D0C7A31-009A-4A19-9663-0F99791FAAC5}.UI Test|iPhoneSimulator.Build.0 = Release|Win32
 		{0D0C7A31-009A-4A19-9663-0F99791FAAC5}.UI Test|iPhoneSimulator.Deploy.0 = Release|Win32
-		{0D0C7A31-009A-4A19-9663-0F99791FAAC5}.UI Test|x64.ActiveCfg = Debug|x64
+		{0D0C7A31-009A-4A19-9663-0F99791FAAC5}.UI Test|x64.ActiveCfg = Release|Win32
```
Comment (willyt150) [inline src/SmartHotel.Clients.sln]: This one too please?
